lib/membrane_http_adaptive_stream/manifest.ex

```diff
@@ -120,4 +120,9 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest do
   def all_segments_per_track(%__MODULE__{} = manifest) do
     Map.new(manifest.tracks, fn {track_id, track} -> {track_id, Track.all_segments(track)} end)
   end
+
+  @spec all_header_per_track(t()) :: %{optional(track_id :: term()) => String.t()}
+  def all_header_per_track(%__MODULE__{} = manifest) do
```

Comment:
what does 'all header per track' mean?

Reply:
There is one header per right now. I will change the name of this function.